README.md

```diff
@@ -21,10 +21,55 @@ other components of OpenShift via support for a baremetal platform type.
 
 # Instructions
 
-## Configuration
+## Preparation
+
+Considering that this is a new install on a clean OS, the next tasks should be performed prior the installation:
+
+1. Enable passwordless sudo for the current user
+
+    `echo "$USER  ALL=(ALL) NOPASSWD: ALL" >/etc/sudoers.d/${USER}`
+
+2. In case of RHEL, invoke `subscription-manager` in order to `register` and `attach` the subscription
+
+3. Install new packages
+
+    `sudo dnf update`
+    `sudo dnf install git make wget tmux jq`
```

Comment:
tmux or screen are optional, I think both come from EPEL

Reply:
True, it's opinionated.
Anyone else thinks that it should be removed?

Reply:
Let's do the bare minimum, +1 to remove tmux